src/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -71,31 +71,36 @@
 
     private OwnCloudVersion mVersion = null;
     
+    private boolean mUseNextcloudUserAgent = false;
+    
     /**
      * Constructor
      */
-    public OwnCloudClient(Uri baseUri, HttpConnectionManager connectionMgr) {
+    public OwnCloudClient(Uri baseUri, HttpConnectionManager connectionMgr, boolean useNextcloudUserAgent) {
         super(connectionMgr);
         
         if (baseUri == null) {
         	throw new IllegalArgumentException("Parameter 'baseUri' cannot be NULL");
         }
         mBaseUri = baseUri;
+        mUseNextcloudUserAgent = useNextcloudUserAgent;
         
         mInstanceNumber = sIntanceCounter++;
         Log_OC.d(TAG + " #" + mInstanceNumber, "Creating OwnCloudClient");
 
-        String userAgent = OwnCloudClientManagerFactory.getUserAgent();
-        getParams().setParameter(HttpMethodParams.USER_AGENT, userAgent);
-        getParams().setParameter(
-        		PARAM_PROTOCOL_VERSION,
-        		HttpVersion.HTTP_1_1);
+        String userAgent;
+
+        if (useNextcloudUserAgent) {
+            userAgent = OwnCloudClientManagerFactory.getNextcloudUserAgent();
+        } else {
+            userAgent = OwnCloudClientManagerFactory.getUserAgent();
+        }
         
-        getParams().setCookiePolicy(
-        		CookiePolicy.IGNORE_COOKIES);
-        getParams().setParameter(
-        		PARAM_SINGLE_COOKIE_HEADER, 			// to avoid problems with some web servers
-        		PARAM_SINGLE_COOKIE_HEADER_VALUE);
+        getParams().setParameter(HttpMethodParams.USER_AGENT, userAgent);
+        getParams().setParameter(PARAM_PROTOCOL_VERSION, HttpVersion.HTTP_1_1);
```

Comment:
Why 1.1?

Reply:
I have no idea ;-)
It is on master all the time: https://github.com/nextcloud/android-library/blob/master/src/com/owncloud/android/lib/common/OwnCloudClient.java#L92